gunicorn/util.py

```diff
@@ -532,3 +532,15 @@ def warn(msg):
 
     print("!!!\n", file=sys.stderr)
     sys.stderr.flush()
+
+
+def make_fail_app(msg):
+
+    def app(environ, start_response):
+        start_response("500 Internal Server Error", [
+            ("Content-Type", "text/plain"),
+            ("Content-Length", str(len(msg)))
+        ])
+        return iter([msg])
```

Comment:
oh I copied it from the example on http://gunicorn.org/ , and you're right, `[msg]` is enough